docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -17,18 +17,26 @@ When using the DMT, bear the following in mind:
 ** The source cluster must be running and populated with required data
 ** The application must be using the source cluster successfully before migration
 ** Before starting the migration, remember to shutdown any applications that are using the source cluster
-** If migrating Replicated map metadata, you must add the custom JAR to your class path and restart the cluster. If this is not done, the Replicated map metadata is not migrated. For further information on adding the custom JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
+** For certain versions, you must add the provided extension JAR to your class path and restart the cluster if you want to migrate Replicated map metadata and/or CP data structures. For further information on the affected versions and use of the extension JAR, see the xref:migrate:dmt-source-cluster-extension.adoc[] topic
++
+If needed, but not done, Replicated map metadata and/or CP data structure are not migrated. For further information on adding the extension JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
 
 * *Target Cluster:*
 
 ** The target cluster must be set up
 +
 Any populated data structures that already exist on the target cluster are not migrated; however, if the existing data structure is empty, it is migrated.
 
-** The target cluster must be running on either Enterprise Edition, version 5.3.2 or later, or {hazelcast-cloud}
+** The target cluster must be running on oneof the following:
+
+*** If you don't want to migrate ReplicatedMap metadata, Enterprise Edition version 5.3.2 or later. You can disable migration of Replicated map metadata as described in xref:migrate:dmt-connection-config.adoc#replicatedmap-metadata-migration[ReplicatedMap Metadata Migration]
```

Comment:
5.3.2 is for the IMap metadata migration, (which can't be disabled by an option), not for ReplicatedMap